diagnostic.md

```diff
@@ -19,6 +21,7 @@ Record your responses inside the fenced code blocks below each question.
 
     ```md
     <!-- your response here -->
+    The parent files are edited inorder to display the componants content
```

Comment:
Can you expand on that?  In general, the `template.hbs` is responsible for markup, while the `component.js` is responsible for presentation and behavior.  Now that you've used Ember a little more do you feel like you understand the what each does?